src/App.css

```diff
@@ -1,26 +1,240 @@
-:root {
+:root[data-theme="light"] {
   --primary-color: #1a73e8;
   --hover-primary: #1557b0;
   --background-color: #f8f9fa;
   --column-background: #ffffff;
   --border-color: #dadce0;
   --text-primary: #202124;
   --text-secondary: #5f6368;
+  --priority-high: #dc3545;
+  --priority-medium: #ffc107;
+  --priority-low: #28a745;
+  --navbar-bg: #ffffff;
+  --task-bg: #ffffff;
+  --hover-bg: rgba(0, 0, 0, 0.05);
+}
+
+:root[data-theme="dark"] {
+  --primary-color: #7C3AED;
+  --hover-primary: #6D28D9;
+  --background-color: #0F0F11;
+  --column-background: #1A1A1D;
+  --border-color: #27272A;
+  --text-primary: #FFFFFF;
+  --text-secondary: #A1A1AA;
+  --priority-high: #dc3545;
+  --priority-medium: #ffc107;
+  --priority-low: #28a745;
+  --navbar-bg: #18181B;
+  --task-bg: #27272A;
+  --hover-bg: rgba(255, 255, 255, 0.05);
+}
+
+/* Navbar Styles */
+.navbar-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  z-index: 1000;
+  background-color: var(--navbar-bg);
+  border-bottom: 1px solid var(--border-color);
+  transition: background-color 0.3s ease;
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
transition: background-color 0.3s ease;
```
The transition property using 'all' can impact performance as it forces the browser to animate all animatable properties. Specify only the properties that need to transition.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;